mne/report.py

```diff
@@ -772,7 +774,7 @@ def add_section(self, figs, captions, section='custom'):
                                          section=section)
 
     def add_figs_to_section(self, figs, captions, section='custom',
-                            scale=None, image_format='png'):
+                            scale=None, image_format='png', textbox=None):
```

Comment:
I'm +0.5 on that. Can you put your text in a box so that we can call it textbox?